web/ui/templates/status.html

```diff
@@ -101,5 +95,26 @@ <h2 id="alertmanagers">Alertmanagers</h2>
       </tbody>
     </table>
 
+    <h2 id="headstatus">Head Stats</h2>
+    <table class="table table-sm table-bordered table-striped table-hover">
+      <tbody>
+        <tr>
+           <th>Number Of Series </th>
+           <th>Number of Chunks</th>
+           <th>Current Max Time</th>
+           <th>Current Min Time</th>
+        </tr>
+        <tr>
+          <td scope="row">{{ .NumSeries}}</th>
+          <td>{{.ChunkCount}}</td>  
+          <td>{{ .MaxTime}}</td>
+          <td>{{ .MinTime}}</td>
+        </tr>
+      </tbody>
+    </table>
+    <input class="btn btn-primary" type="submit" value="Query Cardinality Stats" id="get_stats"></div>
```

Comment:
Is the button necessary here or should the query just be done on page load? Since the query is cached it won't update if pressed frequently anyway.